openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -321,11 +321,12 @@ def test_get_account_empty_string(self):
         legacy_profile.country = ""
         legacy_profile.level_of_education = ""
         legacy_profile.gender = ""
+        legacy_profile.bio = ""
```

Comment:
Why is bio now behaving this way? All the others are fields where you choose from a list of options.